website/docs/r/datacenter.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_datacenter"
+sidebar_current: "docs-vsphere-resource-datacenter"
+description: |-
+  Provides a VMware vSphere datacenter resource. This can be used as the primary container of inventory objects such as hosts and virtual machines.
+---
+
+# vsphere\_datacenter
+
+Provides a VMware vSphere datacenter resource. This can be used as the primary container of inventory objects such as hosts and virtual machines.
+
+No updates are possible.
```

Comment:
Can this sentence be fleshed out a bit? While the context might be clear to us as contributors and engineers it might not necessarily be to someone reading this document.

You could also move this to the argument reference and make it a warning by using the `~>` prefix. Something like:

```
## Argument Reference

~> **NOTE**: Datacenters cannot be changed once they are created. Modifying any of these attributes will force a new resource!
```